css/style.css

```diff
@@ -0,0 +1,296 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap');
+
+/* Global */
+html {
+  font-family: 'Roboto', sans-serif;
+}
+
+header,
+section,
+.articles {
+  width: 100%;
+}
+
+.flex {
+  display: flex;
+}
+
+.none {
+  display: none;
+}
+
+li,
+ul {
+  list-style-type: none;
+  margin-top: 0;
+}
+
+h2 {
+  font-size: 28px;
+}
+
+.grey1 {
+  background-color: #222;
+}
+
+.grey2 {
+  background-color: #444;
+}
+
+.grey3 {
+  background-color: #666;
+}
+
+.grey4 {
+  background-color: #888;
+}
+
+.grey5 {
+  background-color: #aaa;
+}
+
+.height-100 {
+  height: 100px;
+}
+
+.header li,
+.pic_text h1,
+.blue_box,
+.rectangle,
+.center {
+  text-align: center;
+}
+
+.p-b-10 {
+  margin-bottom: 20px;
+}
+
+/* Header */
+.header {
+  padding-left: 0;
+  justify-content: space-around;
+}
+
+.header li {
+  border-radius: 10px;
```

Comment:
You are using this property 7 times in your file, it would be better to create a class for it, agree?